### ISSUE TITLE
Site:installServer doesn't format valet.conf properly

### ISSUE THREAD

**Issue by joke2k** [cpriego/valet-linux#237]
Hi,
I installed following the guide but I was not able to see static files.

`composer global require cpriego/valet-linux`
`valet install`
`valet port 8888`
`valet park`
into my workspace but nginx logs says:

```
019/11/14 22:20:02 [error] 12418#12418: *66 rewrite or internal redirection cycle while processing "/home/joke2k/.config/composer/vendor/cpriego/valet-linux/server.php", client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/home/joke2k/.valet/valet.sock", host: "freshproject.test:8888", referrer: "http://freshproject.test:8888/"
```

for a laravel empty freshproject.

Digging around I found that the `/etc/nginx/sites-available/valet.conf` has not replaced `VALET_STATIC_PREFIX` with the configured one into `server.php`.
All other constants were replaced properly... I don't know why works like that.

Maybe the slashes `location /VALET_STATIC_PREFIX/ ...` so i tryed this code:
```
$str = "location /VALET_STATIC_PREFIX/ ";

echo "STRING: '". $str ."' <br>";
echo "REPLACED: '". str_replace(['VALET_STATIC_PREFIX'], ['omg'], $str) ."'";
```
but it works as aspected printing:
```
STRING: 'location /VALET_STATIC_PREFIX/ '
REPLACED: 'location /omg/ '
```

Now I replaced manually `VALET_STATIC_PREFIX` with `41c270e4-5535-4daa-b23e-c269744c2f45` and everything works like a charm.

**Comment by jmarcher** [comment]
Are you by any change using Linux as a Subsystem of Windows?

**Comment by joke2k** [comment]
> Are you by any change using Linux as a Subsystem of Windows?

No, Ubuntu 19.10 with PHP 7.3

**Comment by jmarcher** [comment]
Thanks for reporting, fixed it on v2.2.2

**Comment by joke2k** [comment]
you're welcome!
Thank you for fast fixing :D
